ee-galleon-pack/src/main/resources/layers/standalone/web-console/layer-spec.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" ?>
+<layer-spec xmlns="urn:jboss:galleon:layer-spec:1.0" name="web-console">
+    <dependencies>
+        <layer name="secure-management"/>
+        <layer name="core-management"/>
+    </dependencies>
+    <packages>
+        <package name="org.jboss.as.console"/>
+        <package name="org.jboss.as.patching"/>
```

Comment:
@yersan Why this?

Reply:
HAL is not prepared to show the "Pathching" menu entry if `/core-service=patching` resource does not exist.

It is a harmless error, we can live with it, the user would see an `"WFLYCTL0030: No resource definition is registered for address [(\"core-service\" => \"patching\")]"`, which indeed is fine. I don't know if we plan to use patching in the future for a bootable jar installation.

I just added it to avoid displaying this error, however, probably the users will receive a different one if they try to patch a bootable jar. The `org.jboss.as.patching` package has no impact on the disk footprint.

What do you prefer to do with this package?
Now I'm more inclined to remove it and live with the error.

Reply:
@yersan IIRC with an RPM installation EAP doesn't install this (because RPM-based EAP is updated using RPM), so HAL should be able to handle its absence gracefully. I'd prefer that HAL corrects this.

Reply:
https://issues.redhat.com/browse/HAL-1706